nengo_gui/examples/tutorial/22-spa-sequence-controlled.py.cfg

```diff
@@ -1,36 +1,36 @@
-_viz_0 = nengo_gui.components.PointerTemplate(model.vision)
-_viz_config[_viz_0].label_visible = 1
+_viz_0 = nengo_gui.components.Pointer(model.vision,target='default')
+_viz_config[_viz_0].label_visible = True
 _viz_config[_viz_0].width = 0.202031
-_viz_config[_viz_0].y = 0.113391
-_viz_config[_viz_0].x = -0.01656
+_viz_config[_viz_0].y = 0.471719
+_viz_config[_viz_0].x = 0.0833095
 _viz_config[_viz_0].show_pairs = 0
 _viz_config[_viz_0].height = 0.287154
-_viz_1 = nengo_gui.components.PointerTemplate(model.memory)
-_viz_config[_viz_1].label_visible = 1
-_viz_config[_viz_1].width = 0.202031
-_viz_config[_viz_1].y = 0.0302324
-_viz_config[_viz_1].x = 1.39929
+_viz_1 = nengo_gui.components.Pointer(model.memory,target='default')
+_viz_config[_viz_1].label_visible = True
+_viz_config[_viz_1].width = 0.260769
+_viz_config[_viz_1].y = 0.418321
+_viz_config[_viz_1].x = 1.6046
 _viz_config[_viz_1].show_pairs = 0
-_viz_config[_viz_1].height = 0.287154
-_viz_2 = nengo_gui.components.ValueTemplate(model.bg.input)
-_viz_config[_viz_2].label_visible = 1
-_viz_config[_viz_2].width = 0.232773
-_viz_config[_viz_2].x = 1.22001
-_viz_config[_viz_2].y = 1.3345
+_viz_config[_viz_1].height = 0.399982
+_viz_2 = nengo_gui.components.Value(model.bg.input)
+_viz_config[_viz_2].label_visible = True
+_viz_config[_viz_2].width = 0.313664
+_viz_config[_viz_2].x = 1.30536
+_viz_config[_viz_2].y = 1.40156
 _viz_config[_viz_2].max_value = 1
 _viz_config[_viz_2].min_value = -1
-_viz_config[_viz_2].height = 0.361262
-_viz_3 = nengo_gui.components.ValueTemplate(model.thalamus.actions.output)
-_viz_config[_viz_3].label_visible = 1
-_viz_config[_viz_3].width = 0.227926
-_viz_config[_viz_3].x = 0.621798
-_viz_config[_viz_3].y = 1.31364
+_viz_config[_viz_2].height = 0.384941
+_viz_3 = nengo_gui.components.Value(model.bg.output)
```

Comment:
This is the wrong output graph.  It should be showing model.thalamus.actions.output